### DISCUSSION TITLE
Improve description of funding

### DISCUSSION THREAD

**Discussion by Zack-83** [datacite/datacite-suggestions#108] [Metadata Schema Suggestions]
### What is the problem that your suggestion solves?

The description of funding includes too few fields, which forces including more independent pieces of information inside the same field. Moreover, the funding acknowledgement text, mandatory for all projects funded by the European Commission, finds no place in DateCite.

### What solution might meet your needs?

We suggest including additional fields in the DataCite Metadata Scheme:

- Executing agency (+ acronym) + identifier [it may be another entity than the primary funder: Volkswagen Foundation, EURAMET...]
- funding program (+ round) (+ subject) [e.g. Horizon Europe 2022 Environment]
- project acronym
- funder acknowledgement text [compulsory for EU-funded projects, e.g.
"This project has been funded within the European EMPIR research programme. EMPIR projects are co-funded by the European Union’s (EU) Horizon 2020 research and innovation programme and the EMPIR participating states."]
- [optional] funding category [values: third-party EU, third-party international, national, regional, commercial]

I also suggest renaming "award title" into "project long title", "award number" into "project funding number" and "award URI" into "project website URL", since "award" sounds like "prize".

### Your name

Giacomo Lanza

### Your organization

Physikalisch-Technische Bundesanstalt (PTB)

### What alternatives have you tried or considered?

compress more information in the same field; abuse other fields e.g. "Remarks"

### What group(s) would benefit from your suggestion?

- [ ] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [X] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [X] Other

### If other group(s), please describe.

Project coordinators reporting their output to the funder or to the executing agency

**Comment by rebeccaringuette** [comment]
I agree that a field needs to be added for submitters to include acknowledgements, especially given requirements to do so. The field could be named creditText to imitate the new related addition to Schema.org. Agree that funding program is useful for others to see where the funding came from (e.g. to potentially apply for the same program in the future).
Renaming existing fields seems unnecessary. The acronyms for the agency or project could easily be handled in the records associated with those items (e.g. the ROR for the agency).

**Reply by Zack-83** [reply]
> The acronyms for the agency or project could easily be handled in the records associated with those items (e.g. the ROR for the agency).

I agree with you not to save redundant information which is/should be provided somewhere else. For agencies it should be no big deal (most should have a ROR, or an ISNI, and they are also less relevant). But for project I am insecure: RAID is still very young, and I suppose that not all funders/agencies maintain a complete, open and persistent online project database. As for long-termed research activities, they are possibly documented only internally within their home institution. So I would better live with some redundancy in the description of project activities, in a similar way to creators/contributors (full name + given and family name + identifier).
